### PR TITLE
fix(tutorial): per DryRun's feedback

### PR DESCRIPTION
**Pull request by percebus**
# Summary

Ironing out some hiccups that arise during the trainig

# Changes in action

Please navigate to the branch directly

https://github.com/percebus/azure-secure-networking-for-devs/blob/fix/tutorial/docs/tutorial/README.md

### CODE REVIEW

File: docs/tutorial/01/spoke.md [hunk 1]
```diff
@@ -82,59 +82,6 @@ Go to the `default` subnet and associate the NSG.
...
-
-Under the "Resource visualizer" you should see something like this:
-
-![Resource visualizer](../../../assets/img/azure/solution/vnets/spoke/vnet/resources/01.png)
-
-Note that the 2 arrows represent the peering between the 2 VNets.
-
-- [x] Hub > Spoke
-- [x] Spoke > Hub
-
```
Comment (percebus) [inline docs/tutorial/01/spoke.md:137]: Moved to `peering.md`

File: docs/tutorial/02/hub/fw.md [hunk 2]
```diff
@@ -5,23 +5,43 @@ Now that all traffic is going through the **Azure Firewall**, we can start addin
 > [!TIP]
 > Everytime you add a rule, the firewall takes like 1+ full minute(s) to apply the rule.
 
+We're going to add **Network Rules Collection**
+
+1. Go to **Settings** > **Rules (Classic)**
+1. Go to second tab **Network Rules Collection**
+1. Click on **+ Add network rule collection**
```
Comment (percebus) [inline docs/tutorial/02/hub/fw.md:12]: Specified **Network Rules Collection**, as noted by Rachel

File: docs/tutorial/02/hub/vm.md [hunk 3]
```diff
@@ -113,7 +113,13 @@ Search for "Application Security Group" in the Azure Portal's Market Place.
 
 - **Name**: `{my-prefix}-hub-{region}-{id}-vm-jump-asg`
 
-Then link the NIC to the ASG.
+Then link the VM to the ASG.
```
Comment (percebus) [inline docs/tutorial/02/hub/vm.md:116]: ASG is linked to VM, NOT to VM's NIC

**Comment by timmyreilly** [comment]
Nice! Great notes!

**Review by jessica-ern — APPROVED** [review]

**Review by timmyreilly — APPROVED** [review]

**Pull request merged**